integration/src/main/java/scripts/client/RawlsClient.java

```diff
@@ -116,9 +133,28 @@ public WorkspaceDetails createTestWorkspace() throws ApiException {
             .attributes(Map.of());
     var workspaceDetails = workspacesApi.createWorkspace(request);
     log.info("created workspace {}", workspaceDetails.getWorkspaceId());
+    ingestData(workspaceDetails.getNamespace(), workspaceDetails.getName());
     return workspaceDetails;
   }
 
+  private void ingestData(String namespace, String name) throws ApiException {
+    for (int i = 1; i <= 15; i++) {
+      Entity entity = new Entity().entityType("sample").name("sample" + i);
+      entity.setAttributes(
+          Map.of(
+              //          "files",
```

Comment:
How do you plan to handle array data in Rawls? If storing and retrieving it works fine, the question then is if you need to property support the `arrayOf()` property in `ColumnModel`. To find that out, you may have to look at the UI code to see if it relies on this value.

If it is needed, the question then is how can you property populate this information in `ColumnModel`. One approach would be to read the first row in the table and use the type of the attribute to generate this value.